A3A/addons/core/functions/CREATE/fn_createVehicleQRFBehaviour.sqf

```diff
@@ -20,40 +20,116 @@
+					};
+            } else {
+                _landPos set [2, 0];
+                _landPosBlacklist pushBack _landPos;
+                if(_roll >= 20) then {
+					[_vehicle, _crewGroup, _cargoGroup, _posDestination, _posOrigin, _landPos] spawn A3A_fnc_combatLanding;
+				} else {
+                    [_vehicle, _cargoGroup, _posDestination, _posOrigin, _crewGroup] spawn A3A_fnc_fastrope;
+				};
+            };
         }
         else
         {
-            if ((typeOf _vehicle) in vehFastRope) then {
-                [_vehicle, _cargoGroup, _posDestination, _posOrigin, _crewGroup] spawn A3A_fnc_fastrope;
+            if (_vtol != "") then {
+                private _roll = random 100;
+					if(_roll >= 40) then {
+						[_vehicle, _cargoGroup, _posDestination, _posOrigin, _crewGroup] spawn A3A_fnc_fastrope;
+					} else {
+                        if(_roll <= 30) then{
+                            [_vehicle, _cargoGroup, _posDestination, _markerOrigin] spawn A3A_fnc_paradrop;
+                        } else {
+                            [_vehicle, _cargoGroup, _posDestination, _markerOrigin, _resPool] spawn SCRT_fnc_common_paradropVehicle;
+                        };
+					};
             } else {
-                [_vehicle, _cargoGroup, _posDestination, _markerOrigin] spawn A3A_fnc_paradrop;
+                private _roll = random 100;
+					if(_roll >= 35) then {
+						[_vehicle, _cargoGroup, _posDestination, _posOrigin, _crewGroup] spawn A3A_fnc_fastrope;
+					} else {
+                        [_vehicle, _cargoGroup, _posDestination, _markerOrigin] spawn A3A_fnc_paradrop;
+					};
             };
         };
     };
     if (_vehType in FactionGet(all,"vehiclesHelisAttack") + FactionGet(all,"vehiclesHelisLightAttack")) exitWith 
-    {
-        //Attack helicopter
-        [_vehicle, _crewGroup, _posDestination] spawn A3A_fnc_attackHeli;
+    {   //Attack helicopter
+        _landPosAttackheli = [_posDestination, 400, 800, 10, 0, 0.12, 0, [], [[0,0,0],[0,0,0]]] call BIS_fnc_findSafePos;
+        private _posOrigin = getMarkerPos _markerOrigin;
+        _posOrigin set [2, 50];
+
+        {
+            if(_x distance2D _landPosAttackheli < 20) exitWith { _landPosAttackheli = [0, 0, 0] };
+        } forEach _landPosBlacklist;
+
+        if (count units _cargoGroup > 3) then {
+            if !(_landPosAttackheli isEqualTo [0,0,0]) then
+            {
+                _landPosAttackheli set [2, 0];
+                _landPosBlacklist pushBack _landPosAttackheli;
+                [_vehicle, _crewGroup, _cargoGroup, _posDestination, _posOrigin, _landPosAttackheli] spawn A3A_fnc_combatLanding;
+            }
+            else
+            {
+                private _roll = random 100;
+				if(_roll >= 20) then {
+					[_vehicle, _cargoGroup, _posDestination, _posOrigin, _crewGroup] spawn A3A_fnc_fastrope;
+				} else {
+                    [_vehicle, _cargoGroup, _posDestination, _markerOrigin] spawn A3A_fnc_paradrop;
+				};
```

Comment:
The devil works hard but your tab key works harder

Reply:
Editing on the laptop, never easy